.github/workflows/ci-dialect_oss.yaml

```diff
@@ -3,18 +3,9 @@
 # # # # # # # # # # # # # # # #
 name: CI - Dialect Tests - Community Edition
 on:
-  pull_request:
-    paths-ignore:
-      - 'doc/**'
-      - 'ops/**'
-  push:
-    branches:
-      - master
-    paths-ignore:
-      - 'doc/**'
-      - 'ops/**'
+  workflow_call:
 concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}-dialect
```

Comment:
This is not good. It means that if dialect is running and you push a new commit, the dialect tests will not be stopped until the new one starts.

Worst case, they run through if the unit tests fail. Correct?